src/app/cli/README.md

```diff
@@ -205,33 +205,105 @@ fdaa:bb::de6
 
 ### Steering dataset
 
-The Commissioner steers joining of new devices by `Steering Data` in the Commissioner dataset. To add a new CCM AE Joiner:
+The Commissioner steers joining of new devices by `Steering Data` in the Commissioner dataset.
 
 ```shell
+> help joiner
 usage:
-joiner enable (meshcop|ae|nmkp) <joiner-eui64>
-joiner enableall (meshcop|ae|nmkp)
+joiner enable (meshcop|ae|nmkp) <joiner-eui64> [<joiner-password>] [<provisioning-url>]
```

Comment:
Use "[]" for optional arguments or "<>" for required arguments.

Reply:
@jwhui  `[]` is used for optional arguments, and `<>` is used to escape a argument name, so user would not take it as a plain string.

Reply:
That seems to mix the meaning of "<>". I would prefer to use bold, italic, or underline to indicate named values.

- https://en.wikipedia.org/wiki/Usage_message

- https://www.ibm.com/support/knowledgecenter/SSZJPZ_8.5.0/com.ibm.swg.im.iis.common.doc/common/command_conventions.html

Reply:
Thanks for the links! The usage here is directly output from the commissioner CLI with a `help joiner` command, I need to verify if `bold, italic and underline` can also be applied to a console.

Reply:
FYI, the `git` command line shares a similar syntax:
```shell
build [master] ⚡  git branch -h
usage: git branch [<options>] [-r | -a] [--merged | --no-merged]
   or: git branch [<options>] [-l] [-f] <branch-name> [<start-point>]
   or: git branch [<options>] [-r] (-d | -D) <branch-name>...
   or: git branch [<options>] (-m | -M) [<old-branch>] <new-branch>
   or: git branch [<options>] (-c | -C) [<old-branch>] <new-branch>
   or: git branch [<options>] [-r | -a] [--points-at]
   or: git branch [<options>] [-r | -a] [--format]
```

Reply:
Interesting. OK, I am fine with it as is.

BTW, I also noticed the way GitHub colorizes shell is a bit strange.

Reply:
yeah, maybe becuase `<`, `>` and `|` are builtin shell operators.